plugins/examples/webrtcsink-stats-server.rs

```diff
@@ -133,6 +133,24 @@ async fn run(args: Args) -> Result<(), Error> {
         .by_name("ws")
         .unwrap();
 
+    ws.connect("encoder-setup", false, |values| {
+        let encoder = values[3].get::<gst::Element>().unwrap();
+
+        info!("Encoder: {}", encoder.factory().unwrap().name());
+
+        let configured = match encoder.factory().unwrap().name().as_str() {
```

Comment:
You maybe want to get the factory (name) only once here, and also not panic if the element has no factory